### PR TITLE
Add parameters.yaml to package_data

### PR DESCRIPTION
**Pull request by ns-rse**
Looking at testing Google Colab as another alternative to Binder, its missing the key `parameters.yaml` file on installation though. This PR adds it.

### CODE REVIEW

File: pgfinder/config/c_diff_monomer_masses.csv
Comment (bobturneruk) [inline pgfinder/config/c_diff_monomer_masses.csv:1]: Why is this file here?

File: pgfinder/config/e_coli_monomer_masses.csv
Comment (bobturneruk) [inline pgfinder/config/e_coli_monomer_masses.csv:1]: Why is this file here?

File: pgfinder_interactive.ipynb [hunk 1]
```diff
@@ -18,6 +18,17 @@
...
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install ipywidgets\n",
+    "!pip install ipysheet\n",
+    "!pip install git+https://github.com/Mesnage-Org/PGFinder.git@ns-rse/colab"
```
Comment (bobturneruk) [inline pgfinder_interactive.ipynb:29]: Would need to change for a live version.

File: setup.cfg [hunk 2]
```diff
@@ -53,6 +53,13 @@ pypi =
   versioneer>=0.26
   wheel
 
+notebooks =
```
Comment (bobturneruk) [inline setup.cfg:56]: good additional option - minimum versions?

File: setup.cfg [hunk 3]
```diff
@@ -69,6 +76,11 @@ omit =
 console_scripts =
   find_pg = pgfinder.find_pg:main
 
+[options.package_data]
+* =
+  config/*.yaml
```
Comment (bobturneruk) [inline setup.cfg:81]: Why does the interactive notebook need this file? Aren't the parameters generated by user input?

**Review by bobturneruk — COMMENTED** [review]
Some comments / questions...

**Comment by ns-rse** [comment]
Sorry this was meant to be a a draft as I thought I'd need it to test this then realised I don't and started tinkering forgetting to close the PR.

Moving the `*.csv` files to within `pgfinder/` directory means that they can be included as part of the `package_data` and are available when the package is installed git `pip install git+https://github.com/Mesnage-Org/pgfinder@ns-rse/colab` (which is where I'm tinkering).

I'm coming full-circle with this, as locally the error is replicated if the Notebook is started _without_ the `jupyter_notebook_config.py`, if that _is_ loaded then the `.ftrs` file will be uploaded.

**Comment by bobturneruk** [comment]
Perhaps mybinder is now ignoring `jupyter_notebook_config.py` for some reason?

**Comment by ns-rse** [comment]
Yes, that's what I'm trying now, I found [this thread](https://discourse.jupyter.org/t/binder-configuration-for-large-ipywidget-file-upload/4662) which suggested placing it in `~/.jupyter/` directory.

Trying that on my branch but Binder capacity dropped today after Google withdrew funding. This reinforces the notion that we try and get things working with JupyterLite which I'll continue investigating/working on.

**Comment by bobturneruk** [comment]
I see whether you're colab or mybinder, you're still reliant on the big G.

**Comment by bobturneruk** [comment]
Fingers cross Microsoft keeps funding GitHub.
